packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx

```diff
@@ -1,13 +1,26 @@
 import React, { SyntheticEvent, useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { Button, ButtonSet, DatePicker, DatePickerInput, Form, Select, SelectItem, TextInput } from '@carbon/react';
+import {
+  Button,
+  ButtonSet,
+  DatePicker,
+  DatePickerInput,
+  Form,
+  Select,
+  SelectItem,
+  TextInput,
+  Stack,
+  FormGroup,
+  Layer,
+} from '@carbon/react';
 import { showNotification, showToast, useSession, useVisit, useLayoutType } from '@openmrs/esm-framework';
 import { DefaultWorkspaceProps } from '@openmrs/esm-patient-common-lib';
 import { savePatientImmunization } from './immunizations.resource';
 import { mapToFHIRImmunizationResource } from './immunization-mapper';
 import { ImmunizationFormData, ImmunizationSequence } from './immunization-domain';
 import { immunizationFormSub } from './immunization-utils';
 import styles from './immunizations-form.scss';
+import { mockPaginatedImmunizations } from '../../../../__mocks__/immunizations.mock';
```

Comment:
I'm not sure this mock mocks anything that is reasonably provided by the backend. We should update this mock to look like, e.g., the result of listing a convenience set, since presumably the list of valid vaccinations comes from a convenience set of concepts.